Add a section on Export Controls (#545)
stackabletech/documentation: modules/ROOT/pages/export.adoc

= Export Control

== USA

> The USA have the https://en.wikipedia.org/wiki/Export_Administration_Regulations[Export Administration Regulations (EAR)] as the primary regulation to control export.

Reviewer: ```suggestion
The USA adopts the https://en.wikipedia.org/wiki/Export_Administration_Regulations[Export Administration Regulations (EAR)] as the primary regulation to control exports.
```

All of our products are outside of the scope of EAR because they fall under the _publicly available_ exemption and do not contain non-standard cryptography.

> NOTE: That means E-Mail notifications to the NSA and BIS are not required and therefore not published here.

Reviewer: ```suggestion
NOTE: That means E-Mail notifications to the NSA and BIS are not required and are therefore not published here.
```

In particular:

> * We are exempt under https://www.ecfr.gov/current/title-15/subtitle-B/chapter-VII/subchapter-C/part-734/section-734.3[EAR 734.3(b)] because we are published according to https://www.ecfr.gov/current/title-15/subtitle-B/chapter-VII/subchapter-C/part-734/section-734.7[EAR 734.7]

Reviewer: ```suggestion
* We are exempt under https://www.ecfr.gov/current/title-15/subtitle-B/chapter-VII/subchapter-C/part-734/section-734.3[EAR 734.3(b)] because we publish according to https://www.ecfr.gov/current/title-15/subtitle-B/chapter-VII/subchapter-C/part-734/section-734.7[EAR 734.7]
```

Author: Thanks for catching this

> * We are exempt under https://www.ecfr.gov/current/title-15/subtitle-B/chapter-VII/subchapter-C/part-734/section-734.7[EAR 742.15(b)] as all our software only includes https://ecfr.io/Title-15/Section-772.1[standard encryption]

Reviewer: ```suggestion
* We are exempt under https://www.ecfr.gov/current/title-15/subtitle-B/chapter-VII/subchapter-C/part-742/section-742.15[EAR 742.15(b)] as our software includes only https://ecfr.io/Title-15/Section-772.1[standard encryption]
```
